duckduckgo/Android
# tracker network leaderboard summary

Asana Issue URL: https://app.asana.com/0/488551667048375/471718733961912

## Description

Adds the network leaderboard summary to the privacy dashboard. 

## Steps to Test this PR:
1. Open the privacy dashboard - "summary not ready" message is displayed
1. Visit http://private.brindy.org.uk/10sites.html
1. Click on each URL 
1. Keep checking the dashboard
1. By the time you get to the end of the list it should show the network summary
